packages/sdk/src/types/runtime/assets/material.ts

```diff
@@ -50,6 +60,31 @@ export class Material extends Asset implements MaterialLike, Patchable<AssetLike
     /** @inheritdoc */
     public get color() { return this._color; }

```

Comment:
>public get color() { return this._color; } [](start = 4, length = 42)

Seems like you'd want to be able to set color too. In the setter it would just copy to the existing color instance, to preserve the observe hooks.

Reply:
Oh, good idea!